src/app/pages/k-12/hero/hero.scss

```diff
@@ -1,6 +1,11 @@
 @import 'variables';
 @import 'mixins';
 
+html,
+body {
+    height: 100%;
+}
```

Comment:
Is this necessary?  If so, it's a global change that will affect every page, so it should probably be moved to `main.scss` and verified that it doesn't break any other pages.